React/Views/RCTRefreshControl.m

```diff
@@ -31,6 +32,18 @@ - (instancetype)init
 - (void)layoutSubviews
 {
   [super layoutSubviews];
+  
+  // UIRefreshControl is managed so avoid changing it directly
+  // This method may break in future versions of iOS (subview structure may change)
+  // And self.transform won't shift title text
+  if(_isInitialRender) {
```

Comment:
what view are we looking for? can we be more certain that it's the right one.
also I would a better explanation as to what view you're looking for so that whomever has to fix this when it breaks knows what to do :)